tests/runtime/performance/test_cli_perf.py

```diff
@@ -96,21 +96,23 @@ def orq_workflow_run(ray_cluster, orq_project_dir):
     yield workflow_id
 
 
-TEST_TIMEOUT = 20
+BASE_TIMEOUT = 1
+TEST_TIMEOUT = 10
+SUBMIT_TIMEOUT = 15
```

Comment:
Question: do submit really takes up to 15 secs? Id expect something closer to 1 sec...
Same question for other commands... 10 sec for getting results? 

Suggestion: Could you add a unit to the timeout? Like BASE_TIMEOUT_SEC so it is clear what units we have here

Reply:
That was a guess based on my (slow) laptop.

Submitting will import a bunch of stuff and anything with workflow runs will talk to Ray 😬 

The actual times on CI were:
```
6.37s call     tests/runtime/performance/test_cli_perf.py::test_orq_submit_workflow_def
4.31s setup    tests/runtime/performance/test_cli_perf.py::test_get_workflow_run
4.10s setup    tests/runtime/performance/test_cli_perf.py::test_orq_submit_workflow_def
3.08s call     tests/runtime/performance/test_cli_perf.py::test_get_workflow_run_results
2.78s teardown tests/runtime/performance/test_cli_perf.py::test_get_workflow_run_results
2.18s call     tests/runtime/performance/test_cli_perf.py::test_get_workflow_run
0.28s call     tests/runtime/performance/test_cli_perf.py::test_orq_help
0.28s call     tests/runtime/performance/test_cli_perf.py::test_orq_invalid
```
wdyt of
```
BASE_TIMEOUT_SEC = 1
TEST_TIMEOUT_SEC = 5
SUBMIT_TIMEOUR_SEC = 8
```